__tests__/project.test.js

```diff
@@ -0,0 +1,41 @@
+const seed = require("../db/seeds/seed");
+const request = require("supertest");
+const app = require("../app");
+const db = require("../db/connection");
+
+const {
+  articleData,
+  commentData,
+  topicData,
+  userData,
+} = require("../db/data/test-data/index");
+
+beforeEach(() => seed({ articleData, commentData, topicData, userData }));
+afterAll(() => {
+  db.end();
+});
+
+describe("/api/topics", () => {
+  test("GET:200 returns an array of all topics", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        body.forEach((topic) => {
```

Comment:
Ideally we should also check length of array if asserting inside a forEach, to ensure we get all topics back.